meson.build

```diff
@@ -3501,6 +3501,19 @@ meson.add_install_script('sh', '-c', 'touch $DESTDIR@0@'.format(prefixdir))
 
 ############################################################
 
+# Ensure that changes to the docs/ directory do not break the
+# basic Github pages build.
+jekyll = find_program('jekyll', required : false)
+if want_tests != 'false' and jekyll.found()
+        test('github-pages',
+             jekyll,
+             args : ['build',
+                     '--source', join_paths(project_source_root, 'docs'),
+                     '--destination', join_paths(project_build_root, '_site')])
```

Comment:
Just a question, why `'_site'`?

Reply:
it's the default name that the `jekyll` tool uses: https://manpages.debian.org/unstable/jekyll/jekyll.1.en.html#OPTIONS

Reply:
Ah, I see. Thank you.

Reply:
It uses `_` to not conflict with other user files. But we could use whatever we want. Or more precisely, anything except `sysvinit` and `var-log` which are currently used,